api/app/signals/apps/services/domain/images.py

```diff
@@ -0,0 +1,60 @@
+import base64
+import io
+import logging
+import os
+
+from django.core.files.storage import default_storage
+from PIL import Image, UnidentifiedImageError
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
To be consistent with all other services these functions should be in a simple class, for example `PDFImageService` or something like that.

Reply:
While not a fan of introducing state / data-less classes I will do so here for consistency's sake.